pkg/restore/client.go

```diff
@@ -1000,3 +1007,50 @@ func (rc *Client) EnableSkipCreateSQL() {
 func (rc *Client) IsSkipCreateSQL() bool {
 	return rc.noSchema
 }
+
+// PreCheckTableClusterIndex checks whether backup tables and existed tables have different cluster index options。
+func (rc *Client) PreCheckTableClusterIndex(
+	tables []*utils.Table,
+	ddlJobs []*model.Job,
+	dom *domain.Domain,
+) error {
+	for _, table := range tables {
+		oldTableInfo, err := rc.GetTableSchema(dom, table.DB.Name, table.Info.Name)
```

Comment:
rest LGTM.
`oldTableInfo` should be `newTableInfo`, just like L403

Reply:
L403 is table info caught from database after table is created.
L1018 is table info before table is created.
So I think they should be different.